chat.py

```diff
@@ -25,27 +27,34 @@ def send_messages_to_chat(projects_by_owner):
     for owner in projects_by_owner.keys():
         user_to_mention = USERS_MAP.get(owner, owner)
         message = "Hey *@{}*, I've noticed you are one of the owners of the following projects:\n\n".format(user_to_mention)
+        send_message_to_this_owner = False
         for project in projects_by_owner.get(owner):
             project_id = project.get('projectId')
             org = ORGS_NAME_MAPPING.get(project.get('parent').get('id'))
             created_days_ago = int(project.get('createdDaysAgo'))
             cost = project.get('costSincePreviousMonth', 0.0)
             currency = project.get('costCurrency', '$')
             emoji = ''
-            if cost > COST_ALERT_THRESHOLD:
-                emoji = ' ' + COST_ALERT_EMOJI
-            message += "- `{}/{}` created `{} days ago`, costing *`{}`* {}.{}\n".format(org, project_id, created_days_ago, cost, currency, emoji)
+            if cost<= COST_BELOW_VALUE_PROJECTS_EXCLUDED:
+                logger.info('- `{}/{}/{}`, will not be in the message, due to its cost being lower than the minimum warning value'.format(owner, project_id))
```

Comment:
This is more like a debug log level. WDYT?

Reply:
this change is done in commit ff21bcdc9826c136fb88af83b9cac9b9b53ddd03f